### PR TITLE
Ruby setup

### PR DESCRIPTION
**Pull request by parhaml**
# Language

Ruby Setup

# Challenge

Three Largest Numbers

I'm sorry, I didn't fork it!

### CODE REVIEW

File: challenges/2022-10-11-blast-from-the-past/setups/ruby/info.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+Run the tests to check your solution by navigating to this directory and running `ruby solution.rb`.
+
+# Requirements
+
+- [Ruby 2.7+](https://www.ruby-lang.org/en/downloads/)
+- Rspec (Test Runner)
+  - `gem install rspec`
+
+`
```
Comment (ericwburden) [inline challenges/2022-10-11-blast-from-the-past/setups/ruby/info.md:11]: Cheeky little dangling backtick...

File: challenges/2022-10-11-blast-from-the-past/setups/ruby/info.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Running Tests
+
+Run the tests to check your solution by navigating to this directory and running `ruby solution.rb`.
+
+# Requirements
+
+- [Ruby 2.7+](https://www.ruby-lang.org/en/downloads/)
+- Rspec (Test Runner)
+  - `gem install rspec`
```
Comment (ericwburden) [inline challenges/2022-10-11-blast-from-the-past/setups/ruby/info.md:9]: I got a permissions error with `gem install rspec`, but `gem install rspec --user-install` worked flawlessly. Do you mind adding that flag?

**Review by ericwburden — CHANGES_REQUESTED** [review]
Minor suggestion for the `info.md` file.

File: challenges/2022-10-11-blast-from-the-past/setups/ruby/info.md (hunk 1 above)
Comment (parhaml) [reply challenges/2022-10-11-blast-from-the-past/setups/ruby/info.md]: Yikes!

File: challenges/2022-10-11-blast-from-the-past/setups/ruby/info.md (hunk 2 above)
Comment (parhaml) [reply challenges/2022-10-11-blast-from-the-past/setups/ruby/info.md:9]: Added it as a second option. Thanks!

Comment (parhaml) [reply challenges/2022-10-11-blast-from-the-past/setups/ruby/info.md:9]: Added it as a second option, thanks!

**Pull request merged**